ocs_ci/deployment/vmware.py

```diff
@@ -129,6 +118,11 @@ def __init__(self):
         self.cluster_launcer_repo_path = os.path.join(
             constants.EXTERNAL_DIR, "v4-scaleup"
         )
+        if version.get_semantic_ocp_version_from_config() >= version.VERSION_4_13:
```

Comment:
You can probably remove this condition and run the following code directly as this condition will be always evaluated to true for the supported versions.

Reply:
Removed